address/signals.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+
+from address.models import Address
+
+
+@receiver(pre_save, sender=Address)
+def populate_formatted(sender, instance, , **kwargs):
```

Comment:
Woops, looks like there's an accidental comma here.

Reply:
Sorry about that 🤦